src/contract_wrappers/tokens/security_token_wrapper/index.ts

```diff
@@ -0,0 +1,7 @@
+/* istanbul ignore file */
+import { SecurityToken_3_0_0, isSecurityToken_3_0_0 } from './3.0.0';
+
+export type SecurityToken = SecurityToken_3_0_0;
+
+export { SecurityToken_3_0_0, isSecurityToken_3_0_0 };
+export { default as SecurityTokenCommon } from './common';
```

Comment:
You should export a "dummy" common class here. See how it's being done in the General Transfer Manager wrapper